.github/workflows/test.yml

```
@@ -0,0 +1,20 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```

nit: I'd prefer explicitly controlling the version over relying on `*-latest`.